src/fields/Fields.ts

```diff
@@ -362,7 +362,6 @@ export function createDvField<O extends BaseOptions>(
     attrs?: { cls?: string, attr?: Record<string, string>, options?: Record<string, string> }
 ): void {
     if (!managedField) return
-    managedField.value = p[managedField.name] || ""
```

Comment:
This can't be removed
At this point `managedField.value` is empty, therefore when passing it to createDvField of a given field, it won't contain any value.
And most of the fields use `managedField.openModal()` that needs `managedField.value` to be set with the current value of the field (if not, the modal will be init with an empty value)

e.g. https://github.com/mdelobelle/metadatamenu/blob/master/src/fields/models/Input.ts#L233

Reply:
Alright got it. You are setting the managedField.value within the fieldModifier function. That looks good to me

Reply:
Yeah, My goal was to make the dvField logic independent of the page structure itself, since once we have the indexed field we already have all the info we need!